src/widgetastic_patternfly4/select.py

```diff
@@ -134,5 +68,70 @@ def read(self):
         """Returns a string of the text of the selected option."""
         return self.browser.text(self.BUTTON_LOCATOR)
 
-    def __repr__(self):
-        return "{}({!r})".format(type(self).__name__, self.locator)
+
+class CheckboxSelect(Select):
+    ITEMS_LOCATOR = ".//label[contains(@class, 'pf-c-select__menu-item')]"
+    ITEM_LOCATOR = f"{ITEMS_LOCATOR}/span[starts-with(normalize-space(.), {{}})]/preceding-sibling::input"  # noqa
+
+    def item_select(self, items, close=True):
+        if not isinstance(items, (list, tuple, set)):
+            items = [items]
+
+        try:
+            for item in items:
+                element = self.item_element(item, close=False)
+                if not self.browser.is_selected(element):
+                    element.click()
+        finally:
+            if close:
+                self.close()
+
+    def item_deselect(self, items):
+        if not isinstance(items, (list, tuple, set)):
+            items = [items]
+
+        try:
+            for item in items:
+                element = self.item_element(item, close=False)
+                if self.browser.is_selected(element):
+                    element.click()
+        finally:
+            self.close()
+
+    def fill(self, items):
+        for item, value in items.items():
+            if value:
+                self.item_select(item, close=False)
+        self.close()
+
+    def read(self):
+        selected = []
+        try:
+            for item in self._get_items():
+                element = self.item_element(item, close=False)
+                selected.append(self.browser.is_selected(element))
+        finally:
+            self.close()
+
+        return selected
+
+    def _get_items(self, close=False):
+        with self.opened():
+            result = [self.browser.text(el) for el in self.browser.elements(self.ITEMS_LOCATOR)]
+
+        if close:
+            self.close()
+
+        return result
+
+    @property
+    def items(self):
+        """Returns a list of all CheckboxSelect items as strings."""
+        items = self._get_items(close=True)
+
+        return items
+
+
+class RadioSelect(CheckboxSelect):
```

Comment:
I don't think the radio select is an official patternfly4 widget ... this class would probably be better off living inside the test plugin itself.